asyncpraw/models/reddit/collections.py

```diff
@@ -331,19 +350,20 @@ def reorder(self, links: List[Union[str, Submission]]):
 
         .. code-block:: python
 
-           collection = reddit.subreddit("SUBREDDIT").collections("some_uuid")
-           current_order = collection.link_ids
-           new_order = reversed(current_order)
-           collection.mod.reorder(new_order)
+            subreddit = await reddit.subreddit("SUBREDDIT")
+            collection = await subreddit.collections("some_uuid")
+            current_order = collection.link_ids
+            new_order = reversed(current_order)
+            await collection.mod.reorder(new_order)
 
         """
-        link_ids = ",".join(self._post_fullname(post) for post in links)
-        self._reddit.post(
+        link_ids = ",".join([await self._post_fullname(post) for post in links])
```

Comment:
I think this would be executed sequentially. is that okay?

Reply:
You're right this doesn't need to asynchronous.